### DISCUSSION TITLE
NAT loopbak 问题：  R68S R22.7.22 无Docker 国际版固件

### DISCUSSION THREAD

**Discussion by leftvalue** [coolsnowwolf/lede#9547] [R68S / 电犀牛专区]
固件 `openwrt-rockchip-R22.8.8-fastrhino-r68s-squashfs-sysupgrade` 的 upnp 存在 `NAT 回流问题`

**Comment by zpfwin** [comment]
是啊，网心云收益只能是以前的四分之一，upnp有问题啊
